packages/preview-middleware-client/src/flp/WorkspaceConnector.ts

```diff
@@ -3,13 +3,33 @@ import ObjectStorageConnector from 'sap/ui/fl/write/api/connectors/ObjectStorage
 import Layer from 'sap/ui/fl/Layer';
 import VersionInfo from 'sap/ui/VersionInfo';
 
-const path = '/preview/api/changes'; 
+const path = '/preview/api/changes';
+interface Change {
+    support?: {
+        generator?: string;
+    };
+}
+
+function getFlexSettings() {
+    let result;
+    const bootstrapConfig = document.getElementById('sap-ui-bootstrap');
+    if (bootstrapConfig) {
+        result = bootstrapConfig.getAttribute('data-open-ux-preview-flex-settings');
+    }
+    return result;
```

Comment:
Why not return parsed value already here?

Reply:
adapted.